### PR TITLE
Fixed E_NOTICE when no 'term' was provided

### PR DESCRIPTION
**Pull request by Quix0r**
This code caused an `E_NOTICE` when no 'term' parameter was provided.

### CODE REVIEW

File: mod/filer.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ function filer_content(App $a)
 		killme();
 	}
 
-	$term = unxmlify(trim($_GET['term']));
+	$term = unxmlify(isset($_GET['term']) ? trim($_GET['term']) : null);
```
Comment (MrPetovan) [inline mod/filer.php]: Later in the code `$term` string length is checked, please initialize with an empty string instead. You can also wrtie it with `defaults()` like
```php
$term = unxmlify(trim(defaults($_GET, 'term', '')));
```

**Comment by Quix0r** [comment]
I have changed my PR according to recommendation (which I was not aware of) and have rebased it against latest changes.

File: mod/filer.php (hunk 1 above)
Comment (Quix0r) [reply mod/filer.php]: That code looks much better than my "bare".

**Comment by MrPetovan** [comment]
Part of #5470

**Pull request merged**